### PR TITLE
Other dependencies p.2

### PR DESCRIPTION
**Pull request by olegbespalov**
From what I checked the changes have nothing really interesting for us, but on the other side there are bugfixes, and at some point, we probably might want to use the future versions, so it's good to keep diff smaller.

* Maintenance update of fatih/color to `v1.13.0` from `v1.12.0`
* Maintenance update of klauspost/compress to `v1.15.1` from `v1.13.6`
* Maintenance update of mattn/go-colorable to `v0.1.12` from `v0.1.9`

Main issue: #2445

### CODE REVIEW

**Review by MStoykov — APPROVED** [review]

File: Dependencies.md [hunk 1]
```diff
@@ -13,10 +13,10 @@ This is mostly so that we keep up with dependencies for which we want new featur
 For example some dependencies that were in the middle of big updates were skipped as they didn't add anything of value at the time, but were being updated day after day seemingly fixing bugs added the previous day. As in that case that wasn't a dependency we wanted to be on top off - we skipped it.
 
 Through the rest of the development cycle dependencies shouldn't be updated unless:
+
 1. required for development
-2. a particular bug fix/feature is really important
-	
-	
+2. a particular bug fix/feature is really important.
+
```
Comment (MStoykov) [inline Dependencies.md:19]: It's really preferable to not have any other but the necessary changes when a dependency is updated.

Did you make this change by accident? If so - I am fine with moving it to another commit in this PR.

Comment (olegbespalov) [reply Dependencies.md:19]: yes, `license/cla` stuck again (and it seems like I'm missing the access to re-run the webhook). I can forse push again with removing this change :see_no_evil:

Comment (codebien) [reply Dependencies.md]: I still see changes on `Dependencies.md`

Comment (MStoykov) [reply Dependencies.md]: Yeah, you are currently removing a line, @olegbespalov . Not a big problem, but still preferable if we don't do it in the commit updating a dependancy

Comment (olegbespalov) [reply Dependencies.md]: I've reverted it, using checkout now, should be no changes

**Review by MStoykov — APPROVED** [review]

**Review by codebien — APPROVED** [review]
